assets/css/style.css

```diff
@@ -115,6 +115,166 @@ a:hover {
   color: black;
 }
 
+.article-item {
+  display: flex;
+  flex-direction: column;
+  flex-basis: auto;
+  justify-content: center;
+  width: 82%;
+  margin: 0 auto;
+  margin-top: 3rem;
+  margin-bottom: 3rem;
+  background-color: white;
+  border: 1px solid #bac0ce;
+  border-radius: 0.7rem;
+  padding: 0.7rem;
+}
+
+.about-me-item {
+  display: flex;
+  flex-direction: column;
+  flex-basis: auto;
```

Comment:
[Optional] For the 'about-me-item', the property `flex-basis: auto` is disabled when the site live due to the container having `box-sizing: border-box` property. Consider removing the `flex-basis: auto` property as it could cause unexpected issues later when you make your site responsive